tests/vcs/git/GitCommitBearTest.py

```diff
@@ -139,6 +139,17 @@ def test_shortlog_checks_shortlog_trailing_period(self):
         self.assertEqual(self.run_uut(shortlog_trailing_period=False), [])
         self.assertEqual(self.run_uut(shortlog_trailing_period=None), [])
 
+    def test_shortlog_checks_imperative(self):
+        self.git_commit("tag: Add shortlog in imperative")
+        self.assertNotIn("Shortlog of HEAD commit isn't imperative mood",
+                         ''.join(self.run_uut()))
+        self.git_commit("Added invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
+        self.git_commit("Adding another invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
```

Comment:
Hey, you know what would be a very simple test for this ? Just do `git log --oneline > /someplace/file.txt` and make a quick for loop checking this ?
If you could check if this works out - I'd be much more confident. (There may be a few wrong things ... Which we can manually identify as mistakes on our side)

Preferably make a paste bin and ass a link in the commit message of this commit ?